pkg/kubectl/cmd/log.go

```diff
@@ -82,8 +82,12 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"log"},
 	}
 	cmd.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
+	cmd.Flags().Bool("timestamps", false, "Include timestamps on each line in the log output")
 	cmd.Flags().Bool("interactive", true, "If true, prompt the user for input when required. Default true.")
 	cmd.Flags().BoolP("previous", "p", false, "If true, print the logs for the previous instance of the container in a pod if it exists.")
+	cmd.Flags().Int("limit-bytes", 0, "Maximum bytes of logs to return. Defaults to no limit.")
+	cmd.Flags().Int("tail", 0, "Lines of recent log file to display. Defaults to showing all log lines.")
```

Comment:
If someone specifies follow, do they also get the existing logs? Assuming they do, since it seems desirable, I could imagine using --tail=0 with --follow. If we're not going to use negative values (e.g., to specify the equivalent of tail +n), we could use -1 to indicate unset.